test/unit/geometry/polygon-tests.js

```diff
@@ -1,218 +1,13 @@
-          "POLYGON ((1.1 3.3, 3 0, 3 6, 1 3), (2 2, 2 1, 1 1, 2 2))",
-        ],
-      ].forEach(function (item) {
-        assert.strictEqual(item[0].toString(), item[1]);
-      });
-    });
-  });
-  describe("#toJSON()", function () {
-    it("should return geo json of the object", function () {
-      [
-        [
-          new Polygon([
-            new Point(1, 3),
-            new Point(3, 1),
-            new Point(3, 6),
-            new Point(1, 3),
-          ]),
-          '{"type":"Polygon","coordinates":[[[1,3],[3,1],[3,6],[1,3]]]}',
-        ],
-        [
-          new Polygon(
-            [
-              new Point(1.1, 3.3),
-              new Point(3, 0),
-              new Point(3, 6),
-              new Point(1, 3),
-            ],
-            [
-              new Point(2, 2),
-              new Point(2, 1),
-              new Point(1, 1),
-              new Point(2, 2),
-            ],
-          ),
-          '{"type":"Polygon","coordinates":[[[1.1,3.3],[3,0],[3,6],[1,3]],[[2,2],[2,1],[1,1],[2,2]]]}',
-        ],
-      ].forEach(function (item) {
-        assert.strictEqual(JSON.stringify(item[0]), item[1]);
-      });
-    });
-  });
-  describe("#fromString()", function () {
-    it("should parse WKT representation", function () {
-      [
-        [
-          "POLYGON ((10 20, 30 40, 30 -40, 10 20))",
-          [[10, 20, 30, 40, 30, -40, 10, 20]],
-        ],
-        [
-          "POLYGON((11.9 20, 30 40, 30 -40, 11.9 20))",
-          [[11.9, 20, 30, 40, 30, -40, 11.9, 20]],
-        ],
-        [
-          "POLYGON ((35 10, 45 45, 15 40, 10 20, 35 10), (20 30.1, 35 35, 30.1 20, 20 30.1))",
-          [
-            [35, 10, 45, 45, 15, 40, 10, 20, 35, 10],
-            [20, 30.1, 35, 35, 30.1, 20, 20, 30.1],
-          ],
-        ],
-      ].forEach(function (item) {
-        const shape = Polygon.fromString(item[0]);
-        const rings = item[1];
-        assert.strictEqual(shape.rings.length, rings.length);
-        rings.forEach(function (ringPoints, ringIndex) {
-          const shapeRing = shape.rings[ringIndex];
-          assert.strictEqual(shapeRing.length, ringPoints.length / 2);
-          for (let i = 0; i < ringPoints.length / 2; i++) {
-            const p = shapeRing[i];
-            assert.strictEqual(p.x, ringPoints[i * 2]);
-            assert.strictEqual(p.y, ringPoints[i * 2 + 1]);
-          }
-        });
-      });
-    });
-    it("should throw TypeError when WKT representation is invalid", function () {
-      [
-        "POLYGON (10 20, 30 40, -30 -10, 10 20)",
-        "POLYGON ((10 20, 30 40, 30 -40, 10))",
-        "POLYGON ((35 10, 45 45, 15 40, 10 20, 35 10),|(20 30.1, 35 35, 30.1 20, 20 30.1))",
-      ].forEach(function (item) {
-        assert.throws(function () {
-          Polygon.fromString(item);
-        }, TypeError);
-      });
-    });
+  it("tests not supported error", function () {
+    assert.throws(function () {
+      new Polygon();
+    }, ReferenceError);
```

Comment:
These tests are OK, but I would check at least in one unit test that the output string of the `ReferenceError` is equal to its expected value.